packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -116,6 +116,13 @@ class PathError extends CommonError {
   const PathError(title, explanation, fixDetails)
       : super(title, explanation, fixDetails);
 
+  // Activity should have deep link enabled flag.
+  static const missingFlag = PathError(
+    'Activity is missing deep link enabled flag',
+    'The activity must have a <meta-data android:name="flutter_deeplinking_enabled" android:value="true" />',
```

Comment:
"The activity must have the following metadata tag: ...".

Also, however this text is used in the widget tree, we should make sure that the text is selectable so that users can copy this and paste it in their app code.

Reply:
In widget tree it's wrapped in a `SelectionArea` inside a  `_CodeCard` , so it is selectable.